apps/web/src/app/profil/[profileSlug]/ProfilePage.tsx

```diff
@@ -0,0 +1,505 @@
+"use client"
```

Comment:
Changes to this file seem unrelated?

Reply:
I don't know why the diff is like this but I just moved `/page.tsx`​ to `/ProfilePage.tsx`​ since you cannot generate metadata for a client component. Nothing changed in this file